settings.gradle

```diff
@@ -1,4 +1,7 @@
-include ':org.eclipse.paho.android.service',':org.eclipse.paho.android.sample',':paho.mqtt.android.example'
+include ':org.eclipse.paho.android.service'
```

Comment:
this change seems to have broken things on windows and on the travis CI build?

Reply:
@icraggs @hannesa2 fyi that this line seems suspicious ^

thx for the library

Reply:
_"Things"_ .... what's broken here, when you split it in separate files ?
Please show us the Windows error.

CI builds properly https://travis-ci.org/github/eclipse/paho.mqtt.android/builds/671150922

<img width="882" alt="image" src="https://user-images.githubusercontent.com/3314607/84103215-b9b4f980-aa12-11ea-8ca0-6bc22e50a08c.png">

Reply:
I tested on Linux:
<img width="543" alt="image" src="https://user-images.githubusercontent.com/3314607/84103950-c0447080-aa14-11ea-947e-09f0c8aabc4d.png">

I tested on macOS:
<img width="616" alt="image" src="https://user-images.githubusercontent.com/3314607/84103930-aefb6400-aa14-11ea-952a-37eb38315d77.png">

Then I tested previous/parent commit `git checkout b5d5e5ef91e846d39a365` to  
_"Move from minSDK 9 to min 14 (#245)"_

The result is the **same error** on both platforms.

### So it has nothing to do with this commit !

Btw, the `master` is in a very bad condition, last previous commit is 3 years old and gradle is `2.x`

Reply:
I recommend the maintainer do merge `develop` into `master` (and give up `develop`) to have one place of truth.
Btw, `develop` builds properly !

Reply:
sounds good on `develop`

i was looking at the Travis CI logs.  the build started breaking once this was merged in:
https://travis-ci.org/github/eclipse/paho.mqtt.android/builds/694740637#L1585

although now that i see it the final error says gateway timeout, so maybe something external and just a coincidence?

```
A problem occurred configuring root project 'paho.mqtt.android'.
> Could not resolve all dependencies for configuration ':classpath'.
   > Could not download httpcore.jar (org.apache.httpcomponents:httpcore:4.1)
      > Could not get resource 'https://jcenter.bintray.com/org/apache/httpcomponents/httpcore/4.1/httpcore-4.1.jar'.
         > Could not GET 'https://jcenter.bintray.com/org/apache/httpcomponents/httpcore/4.1/httpcore-4.1.jar'. Received status code 504 from server: Gateway Time-out
```

thx for the info!

Reply:
In my point of view the master is dead. The new `master` is `develop`.
That's why I recommend to merge `develop` into `master` @icraggs